installers/docker-compose/scripts/setup.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
```

Comment:
We don't have to, but it will be nice to maintain symmetry between the docker and docker compose installers.
One has deploy.sh, the other has setup.sh. Same with env.sh vs setup.sh creating a env file.

Reply:
I come up with this system because `compose` makes use of `.env` file. The deploy file that we have is mostly for printing out things for folks who follow step-by-step, I don't think it's necessary for `compose` as it's all-in-one go.